launcher/src/main/java/org/apache/spark/launcher/JVMRuntimeOptions.java

```diff
@@ -18,14 +18,14 @@
 package org.apache.spark.launcher;
 
 /**
- * This helper class is used to place the all `--add-opens` options
- * required by Spark when using Java 17. `DEFAULT_MODULE_OPTIONS` has added
+ * This helper class is used to place some JVM runtime options(eg: `--add-opens`)
+ * required by Spark when using Java 17. `DEFAULT_OPTIONS` has added
  * `-XX:+IgnoreUnrecognizedVMOptions` to be robust.
  *
  * @since 3.3.0
```

Comment:
If this is a public API, let's not touch it. We can correct the doc comments though.

Reply:
Okay, let me to correct the doc comments in this pr.